android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
```

Comment:
.gitignore these elipse specific file additions

Reply:
Hmmm... not sure where this came from.  Will remove.